STYLE_GUIDE.md

```diff
@@ -683,8 +686,8 @@
 
 ### Generators
 
-  * Use **Gen** suffix when you have Generator version of the same function.
-  * Use **iter** prefix for variables. [[+]](https://docs.python.org/2/library/stdtypes.html#dict.iteritems)
+  * Use `gen` suffix when you have Generator version of the same function.
```

Comment:
I think here gen should be capitalized

Reply:
Yes, and the same would be for `count` or `index`, but I thought it would be more consistent with treating these naming particles independently from where they are placed (whether they are treated as prefixes or suffixes). But oops, I forgot about `sync`.